src/components/custom/_dhero.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { deriv_app_url } from 'common/utility'
+import { localize } from 'components/localization'
+import { Container, Show } from 'components/containers'
+import device from 'themes/device.js'
+import { Header, Modal, useModal } from 'components/elements'
+import SignupModal from 'components/custom/signup-modal'
+import { Button } from 'components/form'
+import { Lottie } from 'components/elements/lottie.js'
+
+const HeroContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    max-width: 58.4rem;
+
+    ${Header} {
+        color: var(--color-white);
+        display: flex;
+        margin-top: 1.6rem;
+    }
+    h4:first-child {
+        margin-top: 0;
+    }
+`
+const StyledHeader = styled(Header)`
+    color: var(--color-white);
+    display: flex;
+    margin-top: 0;
+`
+const Wrapper = styled.div`
+    background-color: var(--color-black);
+    width: 100%;
+    padding: 10rem 0;
+    display: flex;
+    flex-direction: column;
+`
+const TryForFree = styled(Button)`
+    border: 2px solid var(--color-red);
+    font-weight: bold;
+    line-height: 1.43;
+    margin-top: 3.3rem;
```

Comment:
Why is this needed? (except the margin) doesn't the Join us for free button have the same styling as a normal button?

Reply:
fixed